kolibri_explore_plugin/assets/src/views/SearchPage.vue

```diff
@@ -21,7 +21,16 @@
 
       <template v-else>
         <b-container>
-          <EkKeywords :words="keywords" @click="removeKeyword" />
+          <b-row alignH="between">
+            <b-col>
+              <b-form-checkbox v-model="showUnavailable" name="check-show-unavailable" switch>
+                Show unavailable content
```

Comment:
At one point we (Dylan and me) decided to start marking all new UI text as translatable. See how other components use the `{{ $tr('foo') }}` in the template part, and define a `$trs: { ... }` in the script part.